tadasets/rotate.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from .custom_exceptions import NotImplementedError
+
+def get_phi(row):
+    """Helper function for rotating data.
+
+    row: 2D-entry in matrix with shape (?, 2)
+    """
+    return np.arctan2(row[0], row[1])
+
+def rotate_2D(d, angle, centered_around_zero=True):
+    """Rotate a 2-dimensional figure.
+
+    Parameters
+    ============
+
+    d: the 2-dimensional data to rotate
+    angle: the angle (in radians) to rotate the data around 0
+    centered_around_zero: Boolean specifying whether the data is centered around zero.
+    """
+    try:
+        assert d.shape[1] == 2
+    except AssertionError:
+        raise ValueError("Error: data has {} dimensions, but should only be 2. ".format(d.shape[1]))
+
+    if centered_around_zero is False:
```

Comment:
Since this is all new, could you remove the `cenetered_around_zero` stuff?  I'd rather leave this out and just add it once implemented rather than provide an interface without implementation.

Reply:
Yeah, sure! Good point! should I remove the `ValueError` as well?

Reply:
I think so.  Though, if you have a good case for keeping it, I'm all ears.

Reply:
Well, my thoughts around `ValueError` revolved around using this rotation function for other functions as well. I think that is simply my main point; it would ensure the rotation is not incorrectly used for other dimensions than 2. Maybe we should keep it? Although `centered_around_zero`  is probably removeable.

Reply:
Now I removed `centered_around_zero`, but I kept the `ValueError` :)